src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!-- <h1>Product cards</h1> -->
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product-picture"></div>
+
+      <div class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
+
+      <div class="card__stars">
+        <div class="card__stars_stars-block">
+          <div class="card__stars_star"></div>
+          <div class="card__stars_star"></div>
+          <div class="card__stars_star"></div>
+          <div class="card__stars_star"></div>
+          <div class="card__stars_star"></div>
+        </div>
+
+        <div class="card__stars_text">Reviews: 5</div>
+      </div>
```

Comment:
1. it's better to add a little bit more meaningful class names
2. you should use `modifier` to specify an amount of active stars
3. use semantic tags
```suggestion
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="card__reviews">Reviews: 5</p>
      </div>
```

Reply:
done